ci/environment.yml

```diff
@@ -18,6 +18,8 @@ build & push image:
 
 install:
   stage: setup environment
+  variables:
+    ASSET_PREFIX: /suite-web/${CI_BUILD_REF_NAME}/web
```

Comment:
install is a dependency for jobs from different environments, isn't it? so we can't define the ASSET_PREFIX here

Reply:
i'm not really sure what you mean by different env. Install ins only for that specific branch it was run in meaning it crates the file and then the rest of the jobs download them. but it's branch dependent

Reply:
branch dependent, yes, and this ASSET_PREFIX is only valid for feature branches and develop, right? not for codesign branch, because staging and production needs just `ASSET_PREFIX: /web`

Reply:
yes codesign builds run the 'yarn build:libs' in every job